packages/gatsby-source-wordpress/src/models/remoteSchema.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { findTypeName } from "~/steps/create-schema-customization/helpers"
+import { findNamedTypeName } from "~/steps/create-schema-customization/helpers"
```

Comment:
I created a new helper `findNamedTypeName` to recursively find the named type of a complex type. There was previously a `findTypeName` that did the same but in a way that didn't work with the new schema so I refactored all instances of `findTypeName` to the new util